person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  @count = 0
+
+  def self.count
+    @@count
+  end
+
```

Comment:
- [ ] Great work so far but I faced some errors while creating a person object using this person class 😨 Kindly try to make an object using this person class and fix the error which is coming from this @count variable 🤔 to make your code work as expected 👍